src/Place/Canonical/CanonicalService.php

```diff
@@ -94,4 +93,23 @@ private function getOldestPlace(array $placeIds): string
         asort($placesByCreationDate);
         return array_key_first($placesByCreationDate);
     }
+
+    private function getPlaceWithHighestCompleteness(array $placeIds): string
+    {
+        $maxCompleteness = -1;
+        $placeWithMaxCompleteness = '';
+
+        foreach ($placeIds as $placeId) {
+            $jsonDocument = $this->placeRepository->fetch($placeId);
+            $body = $jsonDocument->getBody();
+            $completeness = $body->completeness;
```

Comment:
There are still 129 places without a `completeness`-score, but since none of them are `Museumpas`, a fallback might not be needed.

Reply:
That is really interesting to know, thank you!